shared/src/main/scala/Elements.scala

```diff
@@ -319,17 +347,47 @@ object Elements:
       case a: String => a.toUpperCase()
     }
 
+    val filterElement: Dyad = addElem(
+      Dyad,
+      "F",
+      "Filter by Function | From Base",
+      List("filter", "keep-by", "from-base", "10->b"),
+      "a: fun, b: lst -> Filter b by truthy results of a",
+      "a: lst, b: fun -> Filter a by truthy results of b",
+      "a: num, b: num -> a in base b - list of digits",
+      "a: num, b: str|lst -> a in base with alphabet b",
+    ) {
+      case (a: VFun, b) =>
+        ListHelpers.filter(ListHelpers.makeIterable(b, Some(true)), a)
+      case (a, b: VFun) =>
+        ListHelpers.filter(ListHelpers.makeIterable(a, Some(true)), b)
+    }
+
     val getContextVariableM = addNilad(
       "m",
       "Get Context Variable M",
-      List("get-context-m", "context-m", "c-var-m", "ctx-m", "ctx-secondary"),
+      List(
+        "get-context-m",
+        "context-m",
+        "c-var-m",
+        "ctx-m",
+        "ctx-secondary",
+        "m"
+      ),
       " -> context variable m"
     ) { ctx ?=> ctx.ctxVarSecondary }
 
     val getContextVariableN = addNilad(
       "n",
       "Get Context Variable N",
-      List("get-context-n", "context-n", "c-var-n", "ctx-n", "ctx-primary"),
+      List(
+        "get-context-n",
+        "context-n",
+        "c-var-n",
+        "ctx-n",
+        "ctx-primary",
+        "n"
```

Comment:
Are `n` and `m` are necessary to give as keywords? If someone writes `n` in literate mode, it's as if it's already been sbcsified, right?

Reply:
For some reason I thought they weren't already sbcsified.